index.d.ts

```diff
@@ -6,6 +6,15 @@ type Arrayable<T> = T | T[];
 
 type WarningHandler = (warning: RollupWarning | string) => void;
 
+/**
+ * Since Rollup bundles as ESM, we don't allow compilation to be CJS.
+ * You can instead configure Rollup to build your whole bundle as non-ESM
+ * using the `format` option on `output`.
+ * See: https://github.com/sveltejs/rollup-plugin-svelte/issues/190#issuecomment-930298165
+ */
+type OverridenModuleFormat = "esm"
```

Comment:
we generally only use single quotes, so this line and the next would need to be updated